doc/README.md

```diff
@@ -192,7 +192,7 @@ the graph, controlled by the *nodecount* option.
 
 * **Node Color**:
   * large positive cum values are red.
-  * large negative cum values are green.
+  * large negative cum values are green; negative values are most likely to appear during profile comparison, see [this section](https://github.com/google/pprof/tree/master/doc#comparing-profiles) for details.
```

Comment:
Keep in 80 columns?

Also, should we use a relative link so that it points to the right file when a part of a PR?

Reply:
@nolanmar511 PTAL